price_feed/lib.rs

```diff
@@ -0,0 +1,197 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[ink::contract]
+mod price_feed {
+    use ink::prelude::vec::Vec;
+
+    #[ink(storage)]
+    pub struct PriceFeed {}
+    impl PriceFeed {
+        #[ink(constructor)]
+        pub fn new() -> Self {
+            Self {}
+        }
+        #[ink(message)]
+        pub fn get_coin_info(
+            &mut self,
+            blockchain: [u8; 32],
+            symbol: [u8; 32],
```

Comment:
Let's maybe create type aliases for these two and use them also in l. 22 for example.